### DISCUSSION TITLE
Mysql server Gone away Error - Myloader

### DISCUSSION THREAD

**Discussion by sangithub84** [mydumper/mydumper#1161] [Q&A]
Hi, I am getting below error message when trying to restore a table on Mysql Database. I was able to backup using mydumper but not able to restore. 

Version of myloader is below
`myloader 0.10.5, built against MySQL 5.7.33-36`

Error message while restoring

`Error restoring dbname.testtable from file dbname.testtabl.sql: MySQL server has gone away`

The SQL file is about 250 MB. 
The same was good when I try to restore at db level but when I try to individual table, its failing. Can you please assist. DB instance looks good. I got Wait timeout=86400 and @@max_allowed_packet='16777216'

**Comment by davidducos** [comment]
Hi @sangithub84, 

Why don't you try with a newer version of MyDumper? Take into account that you need to export and import with the same version. 
When you said that the DB looks good, does it mean that the database is up and running al the time? Ah btw, I saw in Azure databases that it is closing the connection for no reason, that is why I had to implement a feature to reconnect and execute of the last statement.
